docs/medical-api/api-reference/patient/bulk-create-patient.mdx

```diff
@@ -0,0 +1,250 @@
+---
+title: "Bulk Patient Create"
+description: "Creates multiple Patients in Metriport at once."
+api: "POST /medical/v1/patient/bulk"
+---
+
+The Bulk Patient Create flow allows you to insert up to 100,000 patients at Metriport in a
+single request/job. We'll also populate the medical data for those patients automatically.
```

Comment:
> allows you to insert up to 100,000 patients at Metriport
Let's separate limits from the endpoint description. Perhaps something like the following:

```
The Bulk Patient Create endpoint allows you create, and kick off Document Queries for, a batch of Patients in a single request.

<Tip>
Currently, a max of 100,000 patients in a batch is supported.
</Tip>
```

Reply:
done